### PR TITLE
[MM-39504] - Remove Prewritten Messages Feature Flag

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
This PR removes `PrewrittenMessages` feature flag and keeps the Pre-written messages within the tips modal

#### Ticket Link
[MM-39504](https://mattermost.atlassian.net/browse/MM-39504)

#### Related Pull Requests

- Has server changes (https://github.com/mattermost/mattermost-server/pull/18968)

#### Release Note

```release-note
NONE

```

### CODE REVIEW

File: components/create_post/create_post.tsx [hunk 1]
```diff
@@ -602,7 +594,7 @@ class CreatePost extends React.PureComponent<Props, State> {
         // Posting a message completes the tip when there are prewritten messages.
         // We do not complete messages in the control group,
         // so as to not alter behavior in the control group as a result of the A/B test code changes.
```
Comment (neallred) [inline components/create_post/create_post.tsx:596]: Can we remove this comment on lines 602-604? I think that without the extra A/B test logic, the code is clear enough on its own.

**Comment by neallred** [comment]
Looks like there's some styles we can clean up in post.scss:
* [The around the input background](https://github.com/mattermost/mattermost-webapp/blob/master/sass/components/_post.scss#L401-L408)
* [The title for the prewritten messages that were displayed around the post box](https://github.com/mattermost/mattermost-webapp/blob/master/sass/components/_post.scss#L464-L469)

**Comment by neallred** [comment]
@esethna @nab-77 is it still useful to track which prewritten message the user selected (i.e. which chip was clicked in the attached picture)? If so, no change needed. If it is no longer useful, we can make [one more small code deletion](https://github.com/mattermost/mattermost-webapp/blob/master/components/create_post/prewritten_chips.tsx#L152-L154) to stop tracking this.
prewritten-chips

**Review by stevemudie — APPROVED** [review]
Send a message prompt is present as expected.

**Review by marianunez — APPROVED** [review]
LGTM, any cypress tests that may need to be updated to reflect the now permanent tutorial tip behavior?

**Comment by AGMETEOR** [comment]
> LGTM, any cypress tests that may need to be updated to reflect the now permanent tutorial tip behavior?

Screenshot 2021-11-15 at 09 53 41

Cypress tests still run fine.

**Review by neallred — APPROVED** [review]
LGTM! :ship:

**Pull request merged**